src/Sample.xoo

```diff
@@ -1,5 +1,7 @@
 public class Sample1 {
 
+  // xoo
+  // xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://localhost:9000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://localhost:9000/project/issues?id=sample&branch=feature1&issues=AWF7dXRMQRfztktLfBvn&open=AWF7dXRMQRfztktLfBvn)

[ISSUE_KEY]: AWF7dXRMQRfztktLfBvn